frontend/components/JoinGraveBtn.tsx

```diff
@@ -50,6 +52,7 @@ export default function JoinGraveBtn({
     setBrowserExtensionControllerAddress,
   ] = useState<string>('');
   const [joiningStep, setJoiningStep] = useState<number>(0);
+  const [leavingStep, setLeavingStep] = useState<number>(-1);
```

Comment:
why not 0 too?

Reply:
Because it is hidden. Step number 0 makes the flow to be displayed